src/main/java/org/junitpioneer/jupiter/EnvironmentVariableUtils.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.Collection;
+import java.util.Map;
+import java.util.function.Consumer;
+
+import org.junit.jupiter.api.extension.ExtensionConfigurationException;
+
+/**
+ * This class modifies the internals of the environment variables map with reflection.
+ * Warning: If your {@link SecurityManager} does not allow modifications, it fails.
+ */
+public class EnvironmentVariableUtils {
```

Comment:
I don't think this should be public API. Was that an accident or on purpose?

Reply:
accident 😳